docs/user/operator-hub.adoc

```diff
@@ -0,0 +1,271 @@
+:source-highlighter: pygments
+
+# odo integration with Operator Hub
+---
+
+When working in experimental mode, odo provides the ability to work with
+link:https://www.openshift.com/learn/topics/operators[Operators] installed on
+the cluster. It allows listing of Operators, creation of services from CRD
+(Custom Resource Definitions) provided by the Operators, printing the YAML
+definition and providing custom YAML definition to start the service from a
+CRD.
+
+[NOTE]
+====
+Installation of Operators is not a part of odo workflow. It is something that
+your OpenShift/Kubernetes cluster administrator should be able to do for you. 
+====
+
+=== Prerequisites
+
+- `odo` is installed.
+- Required Operators are installed in the project/namespace by cluster
+  administrator.
+- Experimental mode is enabled. For every command (other than those for
+  <<create-project,project creation>>) mentioned in this document to work, we
+  first need to enable the experimental mode. This can be done in
+  different ways. Make sure that you have installed latest version of odo and
+  perform any one of the following actions.
+
+* Enable experimental mode in odo preferences:
++
+[source,shell]
+----
+$ odo prefrence set Experimental true
+----
+
+
+* Export the environment variable `ODO_EXPERIMENTAL`:
++
+[source,shell]
+----
+$ export ODO_EXPERIMENTAL=true
+----
+
+* Prefix every command mentioned below with `ODO_EXPERIMENTAL=true`. For
+  example, to list the Operators installed in current project:
++
+[source,shell]
+----
+$ ODO_EXPERIMENTAL=true odo catalog list services
+----
+
+=== [[create-project]]Creating a project
+Create a project to keep your source code, tests, and libraries organized in a
+separate single unit.
+
+1. Log in to the Kubernetes/OpenShift cluster:
++
+[source,shell]
+----
+$ odo login -u developer -p developer
+----
+
+2. Create a project:
++
+[source,shell]
+----
+$ odo project create myproject
+ ✓  Project 'myproject' is ready for use
+ ✓  New project created and now using project : myproject
+----
+
+=== [[list-operators]]Listing the Operators
+
+To list the Operators installed in current project, execute below command:
```

Comment:
`installed in the current project`

Reply:
No need to have stuff like: `execute the command`

See examples such as: https://odo.dev/docs/creating-a-single-component-application-with-odo/

You just need to say: `List the Operators installed in the current project`.

And then show the command.